src/services/api.tsx

```diff
@@ -39,7 +44,109 @@ export const getPatients = async () => {
         if (name[0].given) patient.firstName = name[0].given.join(", ");
         if (name[0].family) patient.lastName = name[0].family;
       }
+      if (identifier) {
+        patient.identifier = identifier
+          .map((e: any) => {
+            return e.value;
+          })
+          .join(", ");
+      }
+
       return patient;
     }
   );
 };
+
+export const getPatientData = (patientId: string) => {
+  /*
+    getPatientData requests for data from Patient resourcce of id patientId
+    return a Patient object.
+  */
+  const getPatientData = async () => {
+    let response = await client.search({
+      type: "Patient",
+      patient: patientId,
+      query: {}
+    });
+    if (!response.data.entry) return; //patient not found
+    const patientData = response.data.entry[0];
+
+    const patient: Patient = {
+      id: patientData.resource.id
+    };
+
+    // Completing patient information with available data
+    if (patientData.resource.identifier) {
+      patient.identifier = patientData.resource.identifier[0];
+    }
+
+    if (patientData.resource.birthDate) {
+      patient.age = getAge(new Date(patientData.resource.birthDate));
+      patient.birthDate =
+        patientData.resource.birthDate +
+        " (" +
+        patient.age.toString() +
```

Comment:
is it useful to add the age here since there is already a patient.age field ?

Reply:
Right, corrected

Reply:
Right, age removed from birthDate attribute and added manually to the patient card